.github/workflows/linux-eic-shell.yml

```diff
@@ -1233,3 +1235,16 @@ jobs:
           issue-number: ${{ github.event.pull_request.number }}
           body-file: capybara_${{ github.event.pull_request.number }}.md
           edit-mode: replace
+
+  cancel-container:
+    if: cancelled() || failure()
+    needs:
+    - trigger-container
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Cancel container
+        run:
+          echo \
+          curl -X POST \
+             -F "token=${{ secrets.EICWEB_CONTAINER_TRIGGER }}" \
+             https://eicweb.phy.anl.gov/api/v4/projects/290/pipelines/${{ needs.trigger-container.outputs.json.id }}/cancel
```

Comment:
Unsure if json field evaluation this way (`.id`) works...

Reply:
Doesn't seem to work.

Reply:
It should work with `fromJSON()` https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#fromjson